pulp_smash/utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging.version import Version
 from pulp_smash.constants import REPOSITORY_PATH, USER_PATH
 from time import sleep
+from urlparse import urljoin
```

Comment:
> Pulp should produce paths that behave correctly with urljoin, and we've had problems in the past with developers making "relative paths" that have a leading slash.

So the benefit here is that, by using urljoin, we ensure that the relative paths given by Pulp do not have a leading slash?